### DISCUSSION TITLE
Element transparency

### DISCUSSION THREAD

**Discussion by MikaelAarras** [Abdelrhman-AK/WinPaletter#195] [Q&A]
WinPaletter does not let me choose transparency effect to buttons and other elements, why so?

**Comment by Abdelrhman-AK** [comment]
Microsoft didn't implement this in Windows UI, so I won't be able to implement this, there is no registry key that does this.

You can try Rectify11 and choose Mica theme, and open application that will be installed (MicaForEveryone) and choose acrylic presets instead of mica. This will give you transparency effect to mostly all Windows elements
